playbooks/ocp-far-edge-vran-deployment.yaml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Trigger ocp-far-edge-vran-deployment-jenkins-job
+  hosts: all
+  remote_user: kni
+  vars_files: jenkins-user-creds.yaml
```

Comment:
Does this file exist?

Reply:
Removed it, I deleted this file since I stored these in OCP CI vault.